src/rpc/mining.cpp

```diff
@@ -260,7 +260,8 @@ UniValue getmininginfo(const JSONRPCRequest& request)
     obj.push_back(Pair("networkhashps",    getnetworkhashps(request)));
     obj.push_back(Pair("pooledtx",         (uint64_t)mempool.size()));
     obj.push_back(Pair("chain",            Params().NetworkIDString()));
-    statsClient.gauge("network.exahashesPerSecond", getnetworkhashps(request).get_real() / 1e18);
```

Comment:
Why not leave it and accept that its very low (zero?) for now? It will increase over time and at some point we'd be happy to already have it here.